manager/manager.go

```diff
@@ -918,11 +918,11 @@ func (m *manager) createContainerLocked(containerName string, watchSource watche
 	if err != nil {
 		return err
 	}
-	devicesCgroupPath, err := handler.GetCgroupPath("devices")
+	memoryCgroupPath, err := handler.GetCgroupPath("memory")
 	if err != nil {
-		klog.Warningf("Error getting devices cgroup path: %v", err)
+		klog.Warningf("Error getting memory cgroup path: %v", err)
 	} else {
-		cont.nvidiaCollector, err = m.nvidiaManager.GetCollector(devicesCgroupPath)
+		cont.nvidiaCollector, err = m.nvidiaManager.GetCollector(memoryCgroupPath)
```

Comment:
I don't think this is what we want to do... The nvidia manager needs the devices cgroup to discover devices.

Reply:
should we skip this part for cgroups v2 since there is no devices controller?

Reply:
I think the way it was written already handles the case where the devices cgroup doesn't exist.

Reply:
I've guarded it with a `if !cgroups.IsCgroup2UnifiedMode() {` otherwise it prints a warning on cgroups v2